catalog/transformer/catalog-transformer-mgmpv2/src/main/java/org/codice/alliance/catalog/mgmp/converter/MgmpConverter.java

```diff
@@ -0,0 +1,1351 @@
+                                    SOUTH_BOUND_LATITUDE_PATH,
+                                    southBoundLatitude);
+                            addBoundingBoxElement(pathValueTracker,
+                                    NORTH_BOUND_LATITUDE_PATH,
+                                    northBoundLatitude);
+                        }
+
+                        geographicElementIndex++;
+
+                    } catch (ParseException e) {
+                        LOGGER.debug("unable to set location", e);
+                    }
+                });
+
+    }
+
+    private void addBoundingBoxElement(XstreamPathValueTracker pathValueTracker, String path,
+            OptionalDouble optionalDouble) {
+        pathValueTracker.add(new Path(Utilities.replaceIndex(path, geographicElementIndex)),
+                String.format(BOUNDING_BOX_FORMAT, optionalDouble.getAsDouble()));
+    }
+
+    private void addGeographicIdentifier(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        Utilities.addMultiValues(pathValueTracker,
+                Utilities.getValues(metacard, Location.COUNTRY_CODE),
+                GMD_COUNTRY_CODE_PATH,
+                geographicElementIndex);
+        Utilities.addMultiValues(pathValueTracker,
+                Utilities.replace(Utilities.getValues(metacard, Location.COUNTRY_CODE),
+                        "ISO3166-1-a3"),
+                GMD_COUNTRY_CODE_SPACE_PATH,
+                geographicElementIndex);
+        geographicElementIndex += Utilities.getValues(metacard, Location.COUNTRY_CODE)
+                .size();
+    }
+
+    private void addMdIdentificationStatus(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        Utilities.addFieldIfString(pathValueTracker,
+                metacard,
+                RESOURCE_STATUS,
+                RESOURCE_STATUS_PATH,
+                (pathValueTracker1, metacard1) -> {
+                    pathValueTracker1.add(new Path(RESOURCE_STATUS_CODE_LIST_PATH),
+                            "http://mod.uk/spatial/codelist/mgmp/2.0/MGMP_ProgressCode");
+                });
+
+    }
+
+    private void addTemporalElements(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+
+        List<String> starts = Utilities.getDateStrings(metacard, DateTime.START);
+        List<String> ends = Utilities.getDateStrings(metacard, DateTime.END);
+
+        if (starts.size() == ends.size()) {
+            for (int i = 0; i < starts.size(); i++) {
+
+                String start = starts.get(i);
+                String end = ends.get(i);
+
+                int elementIndex = i + 1;
+
+                if (start.equals(end)) {
+                    pathValueTracker.add(new Path(Utilities.replaceIndex(
+                            GMD_TEMPORAL_TIME_INSTANT_ID_PATH,
+                            elementIndex)), "GMLID_c1dcde60-9aaf-11e5-a565-0002a5d5c51b");
```

Comment:
We should look into whether these are required or not.  We may be able to omit them